app/assets/javascripts/settings.coffee

```diff
@@ -107,19 +107,86 @@ document.head.appendChild previewStyle
 
 previewTimeout = null
 
+$(document).ready ->
```

Comment:
Actually `$(document).ready` seems to be recommended by `jquery-turbolinks` as it will handle both `ready` and `page:load`?

The issue with this is rooted somewhere different, I believe, but I can't get on why it doesn't work.

I even tried disabling reinitialization with specific checks, but Turbolinks still manages to slip through.

This has been an issue already present on live RS, so I'd also mark that one as a go-to issue for the JavaScript refactor, where we also can consider moving from Turbolinks 2.5 to Turbolinks 5 (!!!), which maybe fixes a lot of our quarrels with it.